frontend/src/component/pages/shared/UserShortLinksSection.spec.tsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+
+import { UserShortLinksSection } from './UserShortLinksSection';
+import { fireEvent, render } from '@testing-library/react';
+import { IPagedShortLinks } from '../../../service/ShortLink.service';
+
+const sampleShortLinks = [
```

Comment:
Let's move data inside each test case so that there is no inter-dependencies. This also make the tests easy to understand.